crossfit/backend/torch/model.py

```diff
@@ -25,16 +26,33 @@
 from crossfit.utils.torch_utils import cleanup_torch_cache, concat_and_pad_tensors
 
 
+class ModelOutputType(Enum):
+    NUMERIC = "numeric"
+    STRING = "string"
+
+
 class Model:
-    def __init__(self, path_or_name: str, max_mem_gb: int = 16, model_output_type: str = "numeric"):
+    def __init__(
+        self,
+        path_or_name: str,
+        max_mem_gb: int = 16,
+        model_output_type: Any = ModelOutputType.NUMERIC,
```

Comment:
nit, but the output_type isn't `Any` right? It's `Union[ModelOutputType, Dict[str, ModelOutputType]`